src/buildercore/bootstrap.py

```diff
@@ -135,9 +143,11 @@ def is_updating(stackname):
         update_msg='Waiting for AWS to finish creating stack ...',
         exception_class=StackTakingALongTimeToComplete
     )
+
     final_stack = core.describe_stack(stackname)
-    events = [(e.resource_status, e.resource_status_reason) for e in final_stack.describe_events()]
-    ensure(final_stack.stack_status in ['CREATE_COMPLETE'],
+    # NOTE: stack.events.all|filter|limit can take 5+ seconds to complete regardless of events returned
```

Comment:
I think it goes through soem pagination

Reply:
yeah, I was testing on new stacks and old stacks and regardless of the number of the number of events or how I limited the number returned and returned per-page it was a very slow call. Thought I should point it out.